rs/http_endpoints/public/src/common.rs

```diff
@@ -246,12 +246,25 @@ impl IntoResponse for CborUserError {
     }
 }
 
-pub(crate) fn validation_error_to_http_error(
+pub(crate) fn validation_error_to_http_error<C: std::fmt::Debug>(
+    request: &HttpRequest<C>,
     message_id: MessageId,
     err: RequestValidationError,
     log: &ReplicaLogger,
 ) -> HttpError {
-    info!(log, "msg_id: {}, err: {}", message_id, err);
+    match err {
+        RequestValidationError::InvalidSignature(_) => {
+            info!(
+                log,
+                "msg_id: {}, err: {}, request: {:?}",
+                message_id,
+                err,
+                request.content()
```

Comment:
Do we have sufficient debug info when just logging `request.content()` rather than `request`?

If we log the entire `request`, we also get the `HttpRequest::auth: Authentication` part, which contains the `signature` and `signer_pubkey` (and `sender_delegation`) for authenticated requests.